x-pack/plugins/infra/public/components/try_it_button.tsx

```diff
@@ -32,6 +32,7 @@ export const TryItButton = ({
   ...props
 }: Props) => {
   const linkProps = useLinkProps({ ...link });
+  const onClickHandler = onClick || linkProps.onClick;
```

Comment:
**suggestion:** this boolean condition will run either one or the other functions, losing the `linkProps.onClick` in case the onClick property is passed. To avoid this scenario and keep the same behaviour performing a client side navigation, you could merge them into a new handler:
```js
const handleClick = (event) => {
  if (linkProps.onClick) linkProps.onClick(event);
  if (onClick) onClick(event);
};
```

Reply:
makes sense to me 👍  will apply the suggestion